it/test/uk/gov/hmrc/nisp/controllers/NIRecordControllerISpec.scala

```diff
@@ -94,6 +98,17 @@ class NIRecordControllerISpec extends AnyWordSpec
          |}
       """.stripMargin)))
 
+    server.stubFor(
+      post(urlMatching("/pertax/authorise"))
+        .willReturn(
+          aResponse()
+            .withStatus(OK)
+            .withBody(Json.stringify(Json.toJson(PertaxAuthResponseModel(
+              ACCESS_GRANTED, "", None, None
+            ))))
+        )
+    )
+
```

Comment:
I think you can now remove the old stub for the old auth from this now since it wont be using the other auth endpoint